docs/transformers/multibyte-text-normalizer.md

```diff
@@ -0,0 +1,23 @@
+<span style="float:right;"><a href="https://github.com/RubixML/RubixML/blob/master/src/Transformers/MultibyteTextNormalizer.php">[source]</a></span>
+
+# Multibyte Text Normalizer
+This transformer converts all multibyte text to lowercase and removes extra whitespace.
```

Comment:
For those of us who have no idea what a multibyte string is, perhaps we can include a sentence about that?